craco.config.cjs

```diff
@@ -81,6 +97,15 @@ module.exports = {
             })
           }
 
+          // Increase the cacheable file size (by the ServiceWorker) so that main.<hash>.js is cached.
+          if (plugin instanceof InjectManifest) {
+            plugin.config.maximumFileSizeToCacheInBytes = 7.5 * 1024 * 1024 // 7.5 MB
```

Comment:
Do you have context why the cacheable size was changed (e.g. performance/memory) and what does that mean for us, if overwritten like this?

Reply:
See https://github.com/Uniswap/interface/pull/6459#discussion_r1180898182